### ISSUE TITLE
Minimum stability issue

### ISSUE THREAD

**Issue by Arakiri** [braseidon/steam-inventory-laravel#4]
Hello!

When I try to install SteamInventory via composer I get this message : 

[InvalidArgumentException]  
  Could not find package braseidon/steam-inventory-laravel at any version for your minimum-stability (stable). Check the package spelling or your minimum-stability

I'm using laravel 5.2 so it must be the origin of the issue.

Any help would be more than welcomed. 
Thanks a lot for your work :)

**Comment by braseidon** [comment]
Ah yes, @Arakiri, I need to update the `composure.json` for Laravel 5.2. Feel free to submit a pull request fixing the issue if you'd like. I'll try to get to this weekend otherwise.

**Comment by ghost** [comment]
I have same problem
(Could not find package braseidon/steam-inventory-laravel at any version for your minimum-stability (stable). Check the package spelling or your minimum-stability)
At Laravel 5.1 :/

**Comment by Arakiri** [comment]
Any news @braseidon ? Thanks a lot

**Comment by vasil7112** [comment]
@Arakiri I had the same issue. Here is my solution (not optimal though). There might be a better one but i started laravel & composer 3 days ago so i don't really know a lot.

I forked the project, added it to my account, updated the composure.json to Laravel 5.2 and then downloaded it as a new package to my laravel application.

@braseidon I am not attempting to steal the project or any of the credits, the project still remains yours 100%. But for any random user who is looking for a quick-fix and just to try out the api, you may check this https://github.com/vasil7112/steam-inventory-laravel

Basically, just do the same steps for installing but instead of braseidon, put vasil7112.
I will not be though supporting the package forever since i am hopeful that braseidon will come back some day ^_^

**Comment by braseidon** [comment]
Should work now. Open an issue if this isn't resolved.
